### ISSUE TITLE
esites Topic 33: Map Old Country Names to Modern Equivalents [DECISION]

### ISSUE THREAD

**Issue by rondlg** [fieldmuseum/EMu-Documentation#33]
DISCUSSION POINTS: We will also need to come up with an agreed method of mapping historic country names and synonyms to their modern equivalents. 

Example lists of such mappings:
https://www.nationsonline.org/oneworld/historical_countrynames.htm
https://en.wikipedia.org/wiki/List_of_country-name_etymologies
https://en.wikipedia.org/wiki/List_of_alternative_country_names

See also 
[Topic 27: Capitalisation and Centroids](https://github.com/fieldmuseum/EMu-Documentation/issues/27)
[Topic 28: West Indies and Caribbean ](https://github.com/fieldmuseum/EMu-Documentation/issues/28)

**Comment by rondlg** [comment]
DECISION: 

After standardizing Country names to the ISO 3166 list as agreed in [Topic 32](https://github.com/fieldmuseum/EMu-Documentation/issues/32) remaining country names will need to be identified as either synonyms or historic and mapped accordingly here for migration to the new sites module: https://docs.google.com/spreadsheets/d/1vjh-QNcr93Jla8YVrfbooLqj5YY6xZF48-yp8cLhvkA

**Comment by rondlg** [comment]
DATA CHANGES: [Data Changes Doc](https://docs.google.com/spreadsheets/d/1gqrciOw-ATgI4MqS5727f0CTvDohP_XJDvfONGpCT0E)

Pre new development: #51
Post development:

**Comment by rondlg** [comment]
HORNBILL ticket submitted
https://customer.hornbill.com/axiell/servicemanager/request/view/IN00113197

Summary
FMNH: esites current name/synonomy functionality does not give prompts/ create synonyms
Description
The synonomy function in our sites module is not creating synonyms as it should.

On the AllClaPTab when we set "Currently accepted" = No and attach the correct name record there is no prompt asking "Was there a boundary change". The record saves but as a result no synonomy history is built up on AllSynTab tab. Also when we set "Currently accepted" = Yes and save, the current name should autofill with an attachment to itself and nothing happens

See irn 1492 / 959
